conda/core/prefix_data.py

```diff
@@ -348,6 +351,59 @@ def validate_name(self, allow_base: bool = False) -> None:
                 f"{PREFIX_NAME_DISALLOWED_CHARS}"
             )
 
+    # endregion
+    # region Metadata
+
+    @property
+    def created(self) -> datetime | None:
+        """
+        Returns the time when the environment was created, as evidenced by the `conda-meta`
+        directory creation time (if available). Falls back to conda-meta/.creation-timestamp.
+        If the environment does not exist or the creation time cannot be identified, returns
+        None.
+        """
+        try:
+            stat = (self.prefix_path / "conda-meta").stat()
+            if not S_ISDIR(stat.st_mode):
+                # check if path is a dir without running stat() again
+                return None
+        except FileNotFoundError:
+            return None
+        else:
+            # On Windows, ctime represents creation time. On other platforms, ctime gives
+            # last metadata change; creation time comes from birthtime.
+            # Birthtime was introduced for Windows in Py312, hence the fallback below.
+            # On Linux, birthtime is only available in kernel 4.11+ via statx and won't be
+            # available in Python until 3.15. The fallback to ctime would be wrong
+            # because in this platform ctime is last metadata change (not creation time!).
+            if on_mac or on_win:
```

Comment:
So, on linux the creation time for any environments created before this change as added will be `None`. Is there a good place to document that caveat of how this works?